### DISCUSSION TITLE
My result list so far

### DISCUSSION THREAD

**Discussion by berger1920** [slabbi/Z80-CPU-Tester#3] [General]
Hi,

Attached are the results of a few of my spare Z80s. I have some others, but installed in computers... I will add them to the list gradually.

A few remarks:

- Due to the frequencies available in the Z80-CPU-Tester, I have modified the clock frequency columns into 4 values: the manufacturer's specification, the test one, the maximum at which the test passes, and a fourth value which is the frequency at which it fails.
- Even if the test passes when overclocked, the values of the registers of port A vary from one try to another, and are random.
- I have checked the power consumption of all the Z80s, and added the values in a column. But I have found that it is not a good idea to do the check with a normal multimeter (on 200mA scale for example) while passing the tests, because these are affected especially if the clock frequency is high. It is better to just do the current consumption measurement, and replace the jumper before passing the tests.

[Results_Z80_CPU_Tester_-_berger.zip](https://github.com/user-attachments/files/16569318/Results_Z80_CPU_Tester_-_berger.zip)

Best,
Gerardo "Berger"

**Comment by slabbi** [comment]
Thanks, added.

**Comment by berger1920** [comment]
Hi,

I have added a few more (MME and SHARP), and corrected the data of the previously checked ones, because I have found that after pressing the NMI button, these tests give random results depending on the clock frequency. So from now on I will always run these "NMI tests" at 2 MHz.

[Results_Z80_CPU_Tester_-_berger.zip](https://github.com/user-attachments/files/16820354/Results_Z80_CPU_Tester_-_berger.zip)

**Comment by slabbi** [comment]
Thx a lot. I will add these asap.
